bindings/src/Capgemini.PowerApps.SpecFlowBindings/Steps/LookupSteps.cs

```diff
@@ -124,5 +124,20 @@ public static void WhenISelectARelatedLookupInTheForm(string lookupName)
             Driver.WaitUntilAvailable(By.XPath(AppElements.Xpath[AppReference.Entity.LookupFieldExistingValue].Replace("[NAME]", lookupName))).Click();
             Driver.WaitForTransaction();
         }
+
+        /// <summary>
+        /// Selects advanced lookup
+        /// </summary>
+        [When(@"I click to perform an advanced lookup")]
+
+        public static void WhenIClickToPerformAnAdvancedLookup()
+
+        {
+            var fieldContainer = Driver.WaitUntilAvailable(By.Name("id-2015"));
+
+            fieldContainer.Click();
+
+            Driver.WaitForTransaction();
+        }
```

Comment:
This change doesn't look related.

Reply:
We are currently reviewing that block of codes

Reply:
Resolved